src/json/federate-role_info.json

```diff
@@ -0,0 +1,83 @@
+{
+	"roles": {
+		"managed": [{
+				"roleName": "ReadOnly",
+				"policyArn": "arn:aws:iam::aws:policy/ReadOnlyAccess",
+				"federate": true
+			},
+			{
+				"roleName": "UserAdmin",
+				"policyArn": "arn:aws:iam::aws:policy/IAMFullAccess",
+				"federate": true
+			},
+			{
+				"roleName": "SysEng",
+				"policyArn": "arn:aws:iam::aws:policy/PowerUserAccess",
+				"federate": true
+			},
+			{
+				"roleName": "DBAdmin",
+				"policyArn": "arn:aws:iam::aws:policy/job-function/DatabaseAdministrator",
+				"federate": true
+			},
+			{
+				"roleName": "FullAdmin",
+				"policyArn": "arn:aws:iam::aws:policy/AdministratorAccess",
+				"federate": true
+			},
+			{
+				"roleName": "DatadogAWSIntegrationRole",
+				"policyArn": "",
+				"federate": false
+			}, {
+				"roleName": "sgas-dev-admin",
```

Comment:
sgas-dev-admin role name should not be hard coded here.

Reply:
address this part in MA-2435